### PR TITLE
[Workers] Rust tutorial revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
do not merge

### CODE REVIEW

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 1]
```diff
@@ -9,41 +9,41 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
 ## Workers Playground
 
-You can test how your Workers function will look when it's deployed by using the preview service, which you can access with the `preview` command:
+You can test how your Workers function will look when it is deployed by using the `preview` command:
 
 ```sh
 rustwasm-markdown-parser $ wrangler preview --watch
 ```
 
 Using the `preview` command will open a browser tab with your Cloudflare Workers function loaded in the Cloudflare preview UI.
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]: @lukeed is there something else here that can replace 'Cloudflare preview UI'? I don't like this wording bc it edges too close to a dashboard feature-like wording. Isn't this just a rendered preview on a browser?

Comment (lukeed) [reply products/workers/src/content/tutorials/hello-world-rust/index.md]: I can't think of a better phrase, but I think all `wrangler preview` instances are now supposed to be `wrangler dev`? /cc @jkup to confirm

Comment (jkup) [reply products/workers/src/content/tutorials/hello-world-rust/index.md]: Yes that's correct! I must have missed this one. FWIW I still used the phrase "preview" in some places but just replaced the command with `dev`

Comment (deadlypants1973) [reply products/workers/src/content/tutorials/hello-world-rust/index.md]: @jkup so are we saying that `preview` needs to be replaced with `dev`? If so, I'm going to change to this:

```suggestion
Using the `dev` command will open a browser tab with your Cloudflare Workers function loaded for you to preview.
```

Comment (deadlypants1973) [reply products/workers/src/content/tutorials/hello-world-rust/index.md]: and also @jkup , should every `preview` in this tutorial be changed to `dev`?

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 2]
```diff
@@ -9,41 +9,41 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
 ```sh
-rustwasm-markdown-parser $ wrangler preview --watch
+rustwasm-markdown-parser $ wrangler dev --watch
 ```
 
-Using the `preview` command will open a browser tab with your Cloudflare Workers function loaded in the Cloudflare preview UI.
+Using the `dev` command will open a browser tab with your Cloudflare Workers function loaded.
 
-The `--watch` flag for `preview` tells Wrangler to watch your Worker project for changes and update the preview tab live with the latest URL. Let's leave Wrangler running in `--watch` mode for now as we continue the tutorial.
+The `--watch` flag for `dev` tells Wrangler to watch your Worker project for changes and update the preview tab live with the latest URL. Leave Wrangler running in `--watch` mode for now as you continue the tutorial.
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]: section needs update

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 3]
```diff
@@ -90,15 +90,15 @@ pub fn parse() -> String {
...
+    // Check that the output is what you expected.
     let expected_html: &str = "<p>Hello world, this is a <del>complicated</del> <em>very simple</em> example.</p>\n";
     assert_eq!(expected_html, &html_output);
 
     format!("\nHTML output:\n{}", &html_output)
 }
 ```
 
-Now we'll update our `worker/worker.js` to use the new code we've written:
+Ugit pdate your `worker/worker.js` to use the new code you have written:
```
Comment (lukeed) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
Update your `worker/worker.js` to use the new code you have written:
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md (hunk 2 above)
Comment (jkup) [reply products/workers/src/content/tutorials/hello-world-rust/index.md]: @deadlypants1973 sorry for the delay on this! `wrangler dev` is always in watch mode so we can just remove this paragraph!

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 4]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
+Wrangler templates are Git repositories. If you want to create your own templates, or use one from the [Template Gallery](/examples), there is a variety of options to help you get started.
 
 ## Workers Playground
 
-You can test how your Workers function will look when it's deployed by using the preview service, which you can access with the `preview` command:
+You can test how your Workers function will execute when it is deployed by using the `dev` command:
 
 ```sh
-rustwasm-markdown-parser $ wrangler preview --watch
+rustwasm-markdown-parser $ wrangler dev
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md:35]: @jkup is this the new correct command?

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 5]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
 ```
 
-Using the `preview` command will open a browser tab with your Cloudflare Workers function loaded in the Cloudflare preview UI.
-
-The `--watch` flag for `preview` tells Wrangler to watch your Worker project for changes and update the preview tab live with the latest URL. Let's leave Wrangler running in `--watch` mode for now as we continue the tutorial.
+Using the `dev` command will open a browser tab with your Cloudflare Workers function loaded.
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
Using the `dev` command will establish a connection between localhost and an edge server that operates your Worker in development.
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 6]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
 
 This creates a directory called `rustwasm-markdown-parser` which you can now `cd` into.
 
-Wrangler templates are just Git repositories, so if you want to create your own templates, or use one from our [Template Gallery](/templates), there's a ton of options to help you get started.
+Wrangler templates are Git repositories. If you want to create your own templates, or use one from the [Template Gallery](/examples), there is a variety of options to help you get started.
 
 ## Workers Playground
 
-You can test how your Workers function will look when it's deployed by using the preview service, which you can access with the `preview` command:
+You can test how your Workers function will execute when it is deployed by using the `dev` command:
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
You can test how your Workers function will execute when it is deployed by using the [`dev` command](/cli-wrangler/commands#dev):
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 7]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
 
 # Hello World in Rust
 
-This tutorial will walk you through the steps of generating, building, previewing, configuring, and publishing
+In this tutorial, you will learn how to generate, build, preview, configure, and publish
 a Rust-generated WebAssembly serverless function that parses Markdown for Cloudflare Workers.
```
Comment (pedrosousa) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]: I'd keep this line at the end of the previous one, without any line break.

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 8]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
+Cloudflare's command-line tool for managing Workers projects, [Wrangler](https://github.com/cloudflare/wrangler), supports various templates — pre-built collections of code that make it easy to get started writing Workers. You will use the [rustwasm-worker template](https://github.com/cloudflare/rustwasm-worker-template/) to start building your project.
 
-In the command line, generate your Workers project by passing in a project name (we'll use rustwasm-markdown-parser), and the template URL to base your project on.
+In the command line, generate your Workers project by passing in a project name and the template URL to base your project on:
 
 ```sh
 ~ $ wrangler generate rustwasm-markdown-parser https://github.com/cloudflare/rustwasm-worker-template/
 ```
 
 This creates a directory called `rustwasm-markdown-parser` which you can now `cd` into.
```
Comment (pedrosousa) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
This command creates a directory called `rustwasm-markdown-parser` which you can now `cd` into.
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 9]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
 
-Wrangler templates are just Git repositories, so if you want to create your own templates, or use one from our [Template Gallery](/templates), there's a ton of options to help you get started.
+Wrangler templates are Git repositories. If you want to create your own templates, or use one from the [Template Gallery](/examples), there is a variety of options to help you get started.
```
Comment (pedrosousa) [inline products/workers/src/content/tutorials/hello-world-rust/index.md:28]: We're saying that users can create their own templates, but it's not obvious where they can learn more about it. Do we have content on how to create our own templates? We could link it from here, saying for example:
```
You can use a template from the [Template Gallery](/examples), which contains a variety of options to help you get started, or [create your own template](/appropriate-link-here).
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 10]
```diff
@@ -9,41 +9,39 @@ import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start
...
-
-The `--watch` flag for `preview` tells Wrangler to watch your Worker project for changes and update the preview tab live with the latest URL. Let's leave Wrangler running in `--watch` mode for now as we continue the tutorial.
+Using the `dev` command will establish a connection between localhost and an edge server that operates your Worker in development.
 
 ## Building
 
-Let's make our Workers function more interesting. We'll pull in a dependency from the `crates.io` ecosystem called `pulldown-cmark`.
-We'll add this to our `Cargo.toml`:
+Begin building your project by pulling in a dependency from the `crates.io` ecosystem called `pulldown-cmark`.
+Add this to your `Cargo.toml` file:
```
Comment (pedrosousa) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
Add the following content to your `Cargo.toml` file:
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 11]
```diff
@@ -52,8 +50,8 @@ We'll add this to our `Cargo.toml`:
 pulldown-cmark = "0.4.0"
 ```
 
-Now we'll leverage the code in the `string-to-string` example from the `pulldown-cmark` GitHub repository. Let's change
-our `src/lib.rs` to look like this:
+Use the code in the `string-to-string` example from the `pulldown-cmark` GitHub repository. Change
+your `src/lib.rs` to look like this:
```
Comment (pedrosousa) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]: This content could appear in the previous line (without a line break).

File: products/workers/src/content/tutorials/hello-world-rust/index.md [hunk 12]
```diff
@@ -90,15 +88,15 @@ pub fn parse() -> String {
...
+    // Check that the output is what you expected.
     let expected_html: &str = "<p>Hello world, this is a <del>complicated</del> <em>very simple</em> example.</p>\n";
     assert_eq!(expected_html, &html_output);
 
     format!("\nHTML output:\n{}", &html_output)
 }
 ```
 
-Now we'll update our `worker/worker.js` to use the new code we've written:
+Update your `worker/worker.js` to use the new code you have written:
```
Comment (pedrosousa) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
Update your `worker/worker.js` to use the new code:
```

**Review by pedrosousa — APPROVED** [review]

File: products/workers/src/content/tutorials/hello-world-rust/index.md (hunk 7 above)
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
In this tutorial, you will learn how to generate, build, preview, configure, and publish a Rust-generated WebAssembly serverless function that parses Markdown for Cloudflare Workers.
```

File: products/workers/src/content/tutorials/hello-world-rust/index.md (hunk 11 above)
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/hello-world-rust/index.md]:
```suggestion
Use the code in the `string-to-string` example from the `pulldown-cmark` GitHub repository. Change your `src/lib.rs` to look like this:
```

**Pull request merged**